clang/lib/Headers/emmintrin.h

```diff
@@ -49,12 +49,10 @@ typedef __bf16 __m128bh __attribute__((__vector_size__(16), __aligned__(16)));
 #endif
 
 /* Define the default attributes for the functions in this file. */
-#define __DEFAULT_FN_ATTRS                                                     \
-  __attribute__((__always_inline__, __nodebug__,                               \
-                 __target__("sse2,no-evex512"), __min_vector_width__(128)))
-#define __DEFAULT_FN_ATTRS_MMX                                                 \
-  __attribute__((__always_inline__, __nodebug__,                               \
-                 __target__("mmx,sse2,no-evex512"), __min_vector_width__(64)))
+#define __DEFAULT_FN_ATTRS __attribute__((__always_inline__, __nodebug__, __target__("sse2,no-evex512"), __min_vector_width__(128)))
```

Comment:
why lose the 80-column?

Reply:
No particular reason; I've reformatted the lines longer than 80 columns (these lines are now clang-format'd).

(Generally I don't worry about this sort of thing, because I just run clang-format on my change. However, this file's current text was so far from clang-format's preferred style that it added a whole lot of noise. So I had decided not to commit the formatted output. But then also didn't think about column widths.)